OpenSim/Common/MarkerData.cpp

```diff
@@ -486,8 +486,9 @@ void MarkerData::readStoFile(const string& aFileName)
     // If the file was written by STOFileAdapter, make the file readable by
     // Storage. Calls below have no effect otherwise.
     std::string tmpFileName{"tmp.sto"};
-    revertToVersionNumber1(aFileName, tmpFileName);
-    addNumRowsNumColumns(tmpFileName, aFileName);
+    bool versionChanged{revertToVersionNumber1(aFileName, tmpFileName)};
+    if(versionChanged)
+        addNumRowsNumColumns(tmpFileName, aFileName);
```

Comment:
If the requirement to `addNumRowsColumns` is a result of reverting the version number (which it seem it is) then why doesn't `revertToVersionNumber1` simply `addNumRowsColumns` itself when it flags `changedVersion = true` above? Then we would not have to expose `addNumRowsNumColumns`.